<launch>
	<!-- Turtlesim Node-->
	<node pkg="turtlesim" type="turtlesim_node" name="sim"/>
	<node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>
	
	<!-- tf broadcaster node-->
	<node pkg="tf_demo" type="tf_broadcaster" name="turtle1_tf_broadcaster" args="/turtle1"/>
	
	<!-- Second broadcaster node-->
	<node pkg="tf_demo" type="tf_broadcaster" name="turtle2_tf_broadcaster" args="/turtle2"/>
	
	<!-- tf listener node-->
	<node pkg="tf_demo" type="tf_listener" name="listener"/>
</launch>
